Dockerfile

```diff
@@ -1,7 +1,28 @@
-FROM ubuntu:latest
+# CHIA BUILD STEP
+FROM python:3.9 AS chia_build
 
-EXPOSE 8555
-EXPOSE 8444
+RUN dpkg-reconfigure -f noninteractive tzdata
```

Comment:
As tzdata is installed and configured in the final image, I assume the reconfigure here is still necessary for the build to run smoothly?

Reply:
This line isn't needed. I left this in by mistake.

Reply:
Update added with commit 1d77db7
Let me know if you wish that I squash to commits.